server/lib/src/valueset/image/mod.rs

```diff
@@ -417,6 +417,27 @@ impl ValueSetT for ValueSetImage {
         Box::new(self.set.iter().map(|image| image.hash_imagevalue()))
     }
 
+    fn to_scim_value(&self) -> Option<ScimValue> {
+        // TODO: This should be a reference to the image URL, not the image itself!
+        // Does this mean we need to pass in the domain / origin so we can render
+        // these URL's correctly?
```

Comment:
if we return a relative url (ie `/foo/bar/ref` instead of `https://frontend_url:port/foo/bar/ref`) then that would work for clients, be valid, and be way simpler...